.github/workflows/identity-and-trust.yml

```diff
@@ -0,0 +1,83 @@
+name: "[Identity & Trust] Google Chats PR Announcer"
+
+on:
+  workflow_dispatch:
+  schedule:
+    # Every morning at 9:30AM UTC (10:30AM BST), Mondays to Fridays
+    # See https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule
+    - cron: "0 8 * * MON-FRI"
+
+jobs:
+  # See https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-defining-outputs-for-a-job
+  setup:
+    runs-on: ubuntu-latest
+    outputs:
+      repos: ${{ steps.main.outputs.repos }}
+    steps:
+      - id: main
+        # Generated by running the following query in the service-catalogue database:
+        # SELECT short_repo_name FROM view_repo_ownership WHERE (github_team_name = 'Identity' OR github_team_name = 'Discussion' OR github_team_name = 'Transparency & Consent') AND NOT archived
+        run: |
+          REPOS=(
+            cmp-market-research
+            consent-management-platform
+            baton
+            transparency-consent-docs
+            baton-lambda-template
+            discussion-api
+            discussion-platform
+            grl
+            hmac-headers
+            identity
+            identity-account-deletion
+            identity-admin
+            salesforce-message-handler
+            hackday-gustodon-bot
+            identity-admin-api
+            identity-data-retention
+            identity-platform
+            identity-processes
+            identity-test-users
+            baton
+            consent-management-platform
+            discussion-avatar
+            discussion-elasticsearch
+            discussion-modtools
+            dynamo-db-switches
+            gatehouse
+            gateway
+            social-sign-in
+          )
+
+          RESULT=""
+          for repo in "${REPOS[@]}"; do
+            RESULT="$RESULT,guardian/$repo"
+          done
+
+          # remove leading ,
+          RESULT="${RESULT:1}"
+          echo "repos=$RESULT" >> $GITHUB_OUTPUT
+  prnouncer:
+    runs-on: ubuntu-latest
+    needs: setup
+    steps:
+      - name: Use GitHub App Token
+        uses: actions/create-github-app-token@v1
+        id: app-token
+        with:
+          # These values are for the GitHub App "Gu PRnouncer config"
+          # See https://github.com/organizations/guardian/settings/apps/gu-prnouncer-config (only accessible by GitHub owners)
+          app-id: ${{ secrets.APP_ID }}
+          private-key: ${{ secrets.PRIVATE_KEY }}
+          owner: guardian
+
+      - uses: guardian/actions-prnouncer@main
+        with:
+          google-webhook-url: ${{ secrets.IDENTITY_AND_TRUST_GOOGLE_WEBHOOK_URL }}
+          github-repositories: ${{ needs.setup.outputs.repos }}
+          github-token: ${{ steps.app-token.outputs.token }} # A GitHub app token generated by the previous step
+
+          # By default, guardian/actions-prnouncer doesn't report PRs from Dependabot. Include them here.
+          github-ignored-users: ""
```

Comment:
Do we possibly want to ignore Dependabot and Scalasteward PRs for now until we deal with our backlog?